loguru/_defaults.py

```diff
@@ -39,9 +40,9 @@ def env(key, type_, default=None):
 LOGURU_FILTER = env("LOGURU_FILTER", str, None)
 LOGURU_LEVEL = env("LOGURU_LEVEL", str, "DEBUG")
 LOGURU_COLORIZE = env("LOGURU_COLORIZE", bool, None)
-LOGURU_SERIALIZE = env("LOGURU_SERIALIZE", bool, False)
+LOGURU_SERIALIZE = env("LOGURU_SERIALIZE", bool, True if HARDENED_BUILD else False)
```

Comment:
We could just say HARDENED_BUILD, but I like the explicit notation of using an if to evaluate the true/false. Feel free to comment on the contrary.

Reply:
I totally agree with you here. Explicitly conversion from `HARDENED_BUILD` to a boolean value is better. :+1: